setup.cfg

```diff
@@ -65,19 +112,14 @@ vagrant =
 windows =
     pywinrm
 
-[pbr]
-skip_authors = True
-skip_changelog = True
-warnerrors = True
-
-[entry_points]
+[options.entry_points]
 console_scripts =
-    molecule = molecule.shell:main
+    molecule = molecule.__main__:main
```

Comment:
Didn't see `molecule.shell` renamed to `molecule.__main__` in this PR?

Reply:
It's not renamed. It's just a common entry point for invoking it `python -m molecule` style. Just wanted to wire it through the same invocation point.